contrib/epee/include/net/net_utils_base.h

```diff
@@ -218,6 +252,10 @@ namespace net_utils
 					return this_ref.template serialize_addr<net::tor_address>(is_store_, stg, hparent_section);
 				case address_type::i2p:
 					return this_ref.template serialize_addr<net::i2p_address>(is_store_, stg, hparent_section);
+#if 0
```

Comment:
Remove?

Reply:
Well, I thought this would make it apparent that it was missing on purpose. This is something I was not sure was best in or out.

Reply:
A short comment would help if the intent was not include.

Reply:
It's not included in the enum now.